Extension/src/LanguageServer/Providers/onTypeFormattingEditProvider.ts

```diff
@@ -14,11 +14,12 @@ export class OnTypeFormattingEditProvider implements vscode.OnTypeFormattingEdit
     }
 
     public async provideOnTypeFormattingEdits(document: vscode.TextDocument, position: vscode.Position, ch: string, options: vscode.FormattingOptions, token: vscode.CancellationToken): Promise<vscode.TextEdit[]> {
+
```

Comment:
Why is there a newline added here? i.e. what newline rule is being followed?

Reply:
Wasnt put there by the formatter, I cleaned it up.